### PR TITLE
CMF-7: CRUD Menu

### PR DESCRIPTION
**Pull request by malavsky**
-  add base entity
- add structure for bundle
- create admin for Menu entity

http://jira.grossum.com/browse/CMF-7

### CODE REVIEW

File: DependencyInjection/GrossumMenuExtension.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        $loader->load('services.yml');
+        $loader->load('classes.yml');
+        $loader->load('admin.yml');
+
+        $this->configureParameterClass($container, $config);
+    }
+
+    /**
+     * @param ContainerBuilder $container
+     * @param $config
```
Comment (WhiteEagle88) [inline DependencyInjection/GrossumMenuExtension.php]: Please, add type for param

File: DependencyInjection/GrossumMenuExtension.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
```
Comment (aivus) [inline DependencyInjection/GrossumMenuExtension.php]: Missed spaces around `.`

File: Entity/BaseMenu.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Grossum\MenuBundle\Entity;
+
+class BaseMenu
+{
+    /**
+     * @var string
+     */
+    private $name;
```
Comment (aivus) [inline Entity/BaseMenu.php]: `protected`?

**Comment by WhiteEagle88** [comment]
Please, add composer.json and check in DI on enabled gedmo

**Comment by malavsky** [comment]
@WhiteEagle88 added composer.json

**Pull request merged**